.ci/dockerfiles/Dockerfile.xpu

```
@@ -0,0 +1,134 @@
+ARG UCX_SRC_DIR=/tmp/ucx_source
+ARG UCX_INSTALL_DIR=/usr
+
+ARG NIXL_SRC_DIR=/workspace/nixl
+
+# Prerequisites for adding external apt repositories
+RUN apt-get update -y && \
+    apt-get install -y --no-install-recommends ca-certificates gnupg software-properties-common wget && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*
+
+# Add Intel oneAPI repo and PPA for GPU support
+RUN wget -O- https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB | gpg --dearmor | tee /usr/share/keyrings/oneapi-archive-keyring.gpg > /dev/null && \
```

Hadolint (2.14.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)